src/study.js

```diff
@@ -80,8 +81,16 @@ export default async function runStudy(devMode) {
             Glean.setUploadEnabled(false);
           }
       });
+      
+      // send the study enrollment ping to the secure analysis environment.
+      const uid = devMode ? "00000000-0000-0000-0000-000000000000" : rally._rallyID;
```

Comment:
It may be worth documenting in the documentation for `rally.id` that the special value `"00000000-0000-0000-0000-000000000000"` will be used if something went wrong with the Rally id.

Reply:
That's a good idea. I may add a couple of cases (test + something went wrong.). Is there a place to put this kind of documentation other than `description`?

Reply:
For Glean projects `description` is currently the best place: it will end up in generated docs and you can use markdown as needed in that field!

Reply:
Perfect, thanks!